com.unity.render-pipelines.universal/Runtime/RendererFeatures/ScreenSpaceShadows.cs

```diff
@@ -34,7 +34,7 @@ public override void Create()
 
             LoadMaterial();
 
-            m_SSShadowsPass.renderPassEvent = RenderPassEvent.AfterRenderingPrePasses;
+            m_SSShadowsPass.renderPassEvent = RenderPassEvent.AfterRenderingGbuffer;
```

Comment:
@phi-lira @ellioman not be able to get access to rendering mode. In forward rendering mode, this pass's event is 'AfterRenderingPrePasses', but to fix flickering in deferred mode, I have to change the event to 'AfterRenderingGbuffer'. this fix will work on both of rendering mode. I wonder if URP provides the rendering mode, RendererFeature will be effective on any rendering mode.

Reply:
Take a look at `ScreenSpaceAmbientOcclusion.cs` 
There we pass the renderer to the `Setup() `function in the pass, and have `isRendererDeferred` check.

Reply:
@ellioman thanks. It will help me :). What if anyone can get the rendering mode outside URP package? for example users are  working on their own renderer feature, they are not be able to get it? I'm just asking you.

Reply:
@Seongdae Good question. I'm not sure why it was decided to put the the rendering mode as Internal, maybe @kaychang-unity can help us there. My feeling is that we did not want to expose it too soon in case of any issues popping up. 

But you're absolutely right, that this might be a pain for users creating their own Rendering Features.

Reply:
@ellioman Thank you :)

Reply:
I made the rendering mode internal as safety measure, initially. It could be made public.